tests/suites/test_suite_pk.data

```diff
@@ -1,7 +1,16 @@
-PK utils: RSA
+PK utils: RSA, 512 bits
 depends_on:MBEDTLS_RSA_C:MBEDTLS_GENPRIME
 pk_utils:MBEDTLS_PK_RSA:512:64:"RSA"
 
+## RSA key generation only supports even bit sizes
+#PK utils: RSA, 511 bits
+#depends_on:MBEDTLS_RSA_C:MBEDTLS_GENPRIME
+#pk_utils:MBEDTLS_PK_RSA:511:64:"RSA"
+#
```

Comment:
It would fail because mbedtls only supports the generation of even-sized RSA keys. We've just reworked the crypto branch to minimize non-PSA-related changes so this chunk of code will go away in the next iteration of this PR.